css/style.css

```diff
@@ -261,6 +251,68 @@ header h2 {
   align-items: center;
 }
 
+.mobile-menu-hide {
+  display: none;
+}
+
+.mobile-menu-show {
```

Comment:
- [ ] In the original design the menu doesn't have a border-radius top, kindly remove it in your design